### PR TITLE
scheduler: log tick information

### PR DESCRIPTION
**Pull request by danielnorberg**

### CODE REVIEW

File: styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java [hunk 1]
```diff
@@ -185,6 +185,11 @@ private void tick0() {
             .sorted(comparingLong(i -> i.runState().timestamp()))
             .collect(toList());
 
+    final String message = String.format("Instances: active=%d, eligible=%d, timedOut=%d",
+        activeStates.size(), eligibleInstances.size(), timedOutInstances.size());
+    LOG.info(message);
+    tracer.getCurrentSpan().addAnnotation(message);
```
Comment (honnix) [inline styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java]: Shall we verify this invocation in test case?

**Comment by honnix** [comment]
minor comment, otherwise LGTM.

File: styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java (hunk 1 above)
Comment (danielnorberg) [reply styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java]: It's not possible to mock `tracer.getCurrentSpan()` 😫

Comment (honnix) [reply styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java]: nvm, i thought that `tracer` was injected.

**Comment by honnix** [comment]
👍

**Pull request merged**